spec/nio/selectables/udp_socket_spec.rb

```diff
@@ -22,7 +22,17 @@
   end
 
   let :writable_subject do
-    pending "come up with a writable UDPSocket example"
```

Comment:
> It was likely pending because of sporadic CI failures.

I really hate it when that happens.  Maybe add a line to output the error type and skip/pending for that (once we see what it is)?